### ISSUE TITLE
Material design Select widget name rendered too low

### ISSUE THREAD

**Issue by cdeil** [holoviz/panel#5620]
With latest releases I find that for the Select widget there's a rendering bug on MacOS browsers.

Code:

```
import panel as pn
pn.extension(design="material")
select = pn.widgets.Select(name="Things", options=["Spam", "Ham", "Jam"])
select.servable()
```

Versions:
```
bokeh 3.3.0
panel 1.3.0a8
```

In Chrome I see this:

<img width="310" alt="Screenshot 2023-10-11 at 22 26 27" src="https://github.com/holoviz/panel/assets/852409/52b18eb5-538b-4bdc-b09e-344e12bd6068">

In Safari I see this:

<img width="312" alt="Screenshot 2023-10-11 at 22 25 56" src="https://github.com/holoviz/panel/assets/852409/9eb067ba-8ade-4c28-831d-30880c43ed12">

I guess this is how it should look:

<img width="652" alt="Screenshot 2023-10-11 at 22 29 07" src="https://github.com/holoviz/panel/assets/852409/fdd0de2e-9113-49e6-8795-49ef3a851415">

**Comment by philippjfr** [comment]
Seems like a regression with latest bokeh. Seems resolved by setting top: -0.5em on `.bk-input-group > label:has(+ select), .bk-input-group > label:has(+ .bk-input-container), .bk-input-group > label:has(+ .bk-spin-wrapper), .bk-input-group > label:has(+ textarea)`

**Comment by philippjfr** [comment]
Here's what that's supposed to look like:

<img width="321" alt="Screen Shot 2023-10-11 at 22 36 28" src="https://github.com/holoviz/panel/assets/1550771/d6e2c6cb-8b98-4855-9f14-8ceb24cbff2f">
